synapse/models/transport.py

```diff
@@ -5,10 +5,44 @@ def getModelDefs(self):
         modl = {
             'types': (
 
+                # TODO is transport:journey a thing?
+
+                ('transport:cargo', ('guid', {}), {
+                    'doc': 'Cargo being carried by a vehicle on a segment.'}),
+
+                ('transport:point', ('str', {'lower': True, 'onespace': True}), {
+                    'doc': 'A departure/arrival point such as an airport gate or train platform.'}),
+
+                ('transport:segment', ('ndef', {'interface': 'transport:segment'}), {
+                    'doc': 'A segment of transportation such as a flight or train ride.'}),
+
+                ('transport:segment:stop', ('guid', {}), {
+                    'interfaces': ('transport:schedule',),
+                    'doc': 'A stop made by a vehicle traveling a segment.'}),
+
+                ('transport:container', ('ndef', {'interface': 'transport:container'}), {
+                    'doc': 'A segment of transportation such as a flight or train ride.'}),
+
+                ('transport:vehicle', ('ndef', {'interface': 'transport:vehicle'}), {
+                    'doc': 'A vehicle such as an air craft or sea vessel.'}),
+
+                ('transport:occupant', ('guid', {}), {
+                    'doc': 'An occupant of a vehicle traveling a segment.'}),
+
+                ('transport:occupant:role:taxonomy', ('taxonomy', {}), {
+                    'interfaces': ('meta:taxonomy',),
+                    'doc': 'A taxonomy of transportation occupant roles.'}),
+
                 ('transport:direction', ('hugenum', {'modulo': 360}), {
                     'doc': 'A direction measured in degrees with 0.0 being true North.'}),
 
+                ('transport:land:vehicle:type:taxonomy', ('taxonomy', {}), {
+                    'interfaces': ('meta:taxonomy',),
+                    'doc': 'A type taxonomy for land vehicles.'}),
+
                 ('transport:land:vehicle', ('guid', {}), {
+                    'interfaces': ('transport:container',),
```

Comment:
Should this inherit transport:vehicle vs. transport:container? (transport:vehicle includes :operator; this would also make it consistent w/the inheritence for transport:sea:vessel and transport:air:craft).